PsfLauncher/StartProcessHelper.h

```diff
@@ -3,18 +3,9 @@
 #include "Logger.h"
 #include <wil\resource.h>
 
-// These two macros don't exist in RS1.  Define them here to prevent build
-// failures when building for RS1.
-#ifndef PROCESS_CREATION_DESKTOP_APP_BREAKAWAY_DISABLE_PROCESS_TREE
-#    define PROCESS_CREATION_DESKTOP_APP_BREAKAWAY_DISABLE_PROCESS_TREE PROCESS_CREATION_DESKTOP_APPX_OVERRIDE
-#endif
-
-#ifndef PROC_THREAD_ATTRIBUTE_DESKTOP_APP_POLICY
-#    define PROC_THREAD_ATTRIBUTE_DESKTOP_APP_POLICY PROC_THREAD_ATTRIBUTE_CHILD_PROCESS_POLICY
-#endif
-
-HRESULT StartProcess(LPCWSTR applicationName, LPWSTR commandLine, LPCWSTR currentDirectory, int cmdShow, bool runInVirtualEnvironment, DWORD timeout)
+HRESULT StartProcess(LPCWSTR applicationName, LPWSTR commandLine, LPCWSTR currentDirectory, int cmdShow, DWORD timeout, LPPROC_THREAD_ATTRIBUTE_LIST attributeList = nullptr)

```

Comment:
Does this really need to be a parameter of the StartProcess function? I don't see the need here since the parameter list created is static.

Reply:
Never mind, I see that it is passed in with breakaway settings for powershell launches.

---
In reply to: [357330588](https://github.com/microsoft/MSIX-PackageSupportFramework/pull/106#discussion_r357330588) [](ancestors = 357330588)